zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/tasks/timeseries/TimeSeriesCompressionTask.java

```diff
@@ -88,11 +90,12 @@ public boolean compressTimeseries(Collection collection, CollectionReader collec
                 collectionWriter.getRoot());
 
         for (TimeseriesCompressionResult failedZipFile : failedZipFiles) {
-            SlackNotification.collectionWarning(collection,
-                    "Failed verification of time series zip file",
-                    new PostMessageField("Attempt", Integer.toString(attempt, 10), true),
-                    new PostMessageField("Zip path", failedZipFile.zipPath.toString(), false)
-            );
+
+            String channel = Root.zebedee.getSlackCollectionAlarmChannel();
+            AttachmentField attemptField = new AttachmentField("Attempt", Integer.toString(attempt, 10), true);
+            AttachmentField zipPath= new AttachmentField("Zip path", failedZipFile.zipPath.toString(), false);
+
+            Root.zebedee.getSlackNotifier().callCollectionWarning(collection, channel, "Failed verification of time series zip file",attemptField, zipPath);
```

Comment:
We can move the channel and the get notifier calls outside of the loop to make it more efficient:

```
String channel = Root.zebedee.getSlackCollectionAlarmChannel();
Notifier notifier =  Root.zebedee.getSlackNotifier();

for (TimeseriesCompressionResult failedZipFile : failedZipFiles) {
    AttachmentField attemptField = new AttachmentField("Attempt", Integer.toString(attempt, 10), true);
    AttachmentField zipPath= new AttachmentField("Zip path", failedZipFile.zipPath.toString(), false);
    
notifier.callCollectionWarning(collection, channel, "Failed verification of time series zip file",attemptField, zipPath);
}
```